consensus/src/network.rs

```diff
@@ -94,8 +94,8 @@ impl NetworkSender {
         let msg = ConsensusMsg::BlockRetrievalRequest(Box::new(retrieval_request.clone()));
         let response_msg = monitor!(
             "block_retrieval",
-            self.network_sender.send_rpc(from, msg, timeout).await?
-        );
+            self.network_sender.send_rpc(from, msg, timeout).await
```

Comment:
There are other places in the code where we are doing this, for example - https://github.com/aptos-labs/aptos-core/blob/94db7a5702ca8376b759b1771dac1dcc0b0ab7ae/consensus/src/epoch_manager.rs#L574. Can you fix them as well?

Reply:
https://github.com/aptos-labs/aptos-core/blob/94db7a5702ca8376b759b1771dac1dcc0b0ab7ae/consensus/src/state_computer.rs#L168 another example

Reply:
updated